Makefile

```diff
@@ -65,6 +65,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/.npminstall:
 ifneq ($(HAS_WEBAPP),)
+	git config --global url."ssh://git@".insteadOf git://
```

Comment:
Was this line intentionally added?

Reply:
@hanzei Yes, I was trying to get the plugin released to community sooner. I'm 0/5 on waiting for the orb to be released